client/layout/masterbar/woo.scss

```diff
@@ -796,6 +796,15 @@ $breakpoint-mobile: 660px;
 		}
 	}
 
+	.login__lost-password-form-wrapper {
+		// Loading lost password form module is very fast, so we don't need to show a visual placeholder, just need to have the space reserved.  Otherwise we will see a flash of content.
```

Comment:
Mega nit
```suggestion
		// Loading lost password form module is very fast, so we don't need to show a visual placeholder, just need to have the space reserved. Otherwise we will see a flash of content.
```